FAQ.md

```diff
@@ -2,7 +2,7 @@
 
 ### What does Noto mean?
 
-When text is rendered by a computer, sometimes characters display as little boxes because your device doesn't have a font that has images for them— these boxes are known as “tofu.”  The name Noto is to convey Google’s goal that users see “__no__ more <b>to</b>fu.”
+When text is rendered by a computer, sometimes characters display as little boxes because your device doesn't have a font that has images for them— these boxes are known as “tofu.”  The name Noto is to convey Google’s goal that users see “__no__ more __to__fu.”
```

Comment:
Hmm, my 2nd commit didn't update this comment. 

But oof, I rebased to squash my 2nd commit that used 2 \* and works:

<img width="147" alt="screen shot 2016-03-25 at 22 35 44" src="https://cloud.githubusercontent.com/assets/261579/14057555/ffffc810-f2d9-11e5-9020-01173f54655a.png">

But there are now a few recent commits by you and Sacha in this PR. Hmm.